ParallelPrimitives/RadixSort.h

```diff
@@ -62,7 +62,7 @@ class RadixSort
 	/// @param device The device.
 	/// @param kernelPath The kernel path.
 	/// @param includeDir The include directory.
-	void compileKernels( oroDevice device, const std::string& kernelPath, const std::string& includeDir ) noexcept;
+	void compileKernels( oroDevice device, OrochiUtils* oroutils, const std::string& kernelPath, const std::string& includeDir ) noexcept;
```

Comment:
(same reason as above) Use reference instead of pointer since this cannot be null and yet we do not check for nullptr in the code.
```suggestion
	void compileKernels( oroDevice device, OrochiUtils& oroutils, const std::string& kernelPath, const std::string& includeDir ) noexcept;
```